core/controllers/suggestion.py

```diff
@@ -148,6 +148,15 @@ def post(self):
 
         self.render_json(self.values)
 
+    @acl_decorators.can_suggest_changes
+    def put(self, suggestion_id):
+        try:
```

Comment:
Are we going to allow suggestion editing for all types of suggestions, If not then please add a flag here!

Reply:
Could you please give me an example for it? I am quite not sure of how to add a flag.